infrastructure/Dockerfile

```diff
@@ -145,6 +149,17 @@ RUN apt-get update \
     || { wget http://apt.dockerproject.org/repo/pool/main/d/docker-engine/docker-engine_${DOCKER_APP_VERSION}_amd64.deb \
     && dpkg -i docker-engine_${DOCKER_APP_VERSION}_amd64.deb; }
 
+# CHRONOS
+#
+WORKDIR /opt
+RUN wget -O chronos-${CHRONOS_APP_VERSION}.tar.gz https://github.com/mesos/chronos/archive/${CHRONOS_APP_VERSION}.tar.gz \
```

Comment:
Why not just:
`CHRONOS_APP_VERSION=2.4.0-0.1.20151007110204.ubuntu1404`
 `apt-get install chronos=${CHRONOS_APP_VERSION}` ?